app/react/V2/Routes/Settings/Thesauri/specs/fixtures.ts

```diff
@@ -0,0 +1,46 @@
+const thesauri = [
+  {
+    _id: 'thesaurus1',
+    name: 'Colors',
+    values: [{ _id: 't1_black', label: 'black', id: 'id1_black' }],
```

Comment:
I don't think thesaurus values have `_id`. Just mentioning in case some test relies on it being there.

Reply:
I'm adding the _id to ensure the values are kept through the sanitization functions, this because #905 & #4053  suggest there could be values with _id, the component is only handling id given that that currently that is the property generated by the endpoint.